test/box/error.test.lua

```diff
@@ -275,3 +275,7 @@ box.schema.func.drop('runtimeerror')
 --
 err = box.error.new('TestType', 'Message arg1: %s. Message arg2: %u', '1', 2)
 err.message
+
+-- gh-4975: errors in init scripts are stripped
+err = box.error.new('TestType', string.rep('a', 10000))
+string.len(err.message)
```

Comment:
Shouldn't we also test that the error message is not actually stripped if an init script raises an error? I usually use `treegen` and `justrun` testing helpers for such tests.

Reply:
Thanks!